Trigger a build to test new build machines

milljm opened the pull request:
DO NOT MERGE

Refs #000

<!--
If this PR is associated with an issue be sure to reference it
by including "refs #<issue>" or "closes #<issue>" (e.g., #closes #1234).

If this PR implements an enhancement that does not have an existing issue, please add the following:
-->

## Reason
<!--Why do you need this feature or what is the enhancement?-->

## Design
<!--A concise description (design) of the enhancement.-->

## Impact
<!--Will the enhancement change existing APIs or add something new?-->


<!--
If this PR implements a bug fix, please create an issue for the bug and reference the issue.
-->

Commit:
Trigger a package build (testing ARM, new machines)

Refs #000

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/975884/) on 90674b1 wanted to post the following:

The following file(s) are changed:

  conda/mpich/conda_build_config.yaml

The conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/975884/) on 90674b1 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/975884/) on 90674b1 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

Commit:
Print env for debugging purposes

Refs #000

moosebuild commented:
Job [Mac stack](https://civet.inl.gov/job/976028/) on 7be92bd : invalidated by @milljm

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/976029/) on 7be92bd wanted to post the following:

The following file(s) are changed:

  conda/mpich/conda_build_config.yaml

The conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/976029/) on 7be92bd wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/976029/) on 7be92bd wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

Commit:
Try to discover CPU count on our own

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/976751/) on 0f3d1a1 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/976751/) on 0f3d1a1 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

Commit:
Use export, unbound variable

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/976783/) on 7146378 wanted to post the following:

The following file(s) are changed:

  conda/mpich/conda_build_config.yaml

The conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/976783/) on 7146378 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

Commit:
include cpu_count script

Commit:
Use MOOSE_JOBS

This will however break default to 2 cores for local builds if one
does not set MOOSE_JOBS...

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/976985/) on ee6a7ae wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

Commit:
Trigger a build... use public channel

The civet recipe has been updated a week ago. Need to verify that
AppleSilicon targets actually use the /public channel and not the
old /arm64 channel.

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/984217/) on ae61948 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/984217/) on ae61948 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

Commit:
Test using conda-forge mpich

Refs #000

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/985121/) on ac9e866 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/985121/) on ac9e866 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Coverage](https://civet.inl.gov/job/985144/) on ac9e866 wanted to post the following:


## Framework coverage

<table>
  <thead>
    <tr>
      <th></th>
      <th>4ab280</th>
      <th colspan=4>#20469 ac9e86</th>
    </tr>
    <tr>
      <th></th>
      <th>Total</th>
      <th>Total</th>
      <th>+/-</th>
      <th>New</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><b>Rate</b></td>
      <td>82.50%</td>
      <td>82.50%</td>
      <td>+0.00%</td>
      <td>-</td>
    </tr>
    <tr>
      <td><b>Hits</b></td>
      <td>73560</td>
      <td>73561</td>
      <td>+1</td>
      <td>0</td>
    </tr>
    <tr>
      <td><b>Misses</b></td>
      <td>15602</td>
      <td>15601</td>
      <td>-1</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

[Diff coverage report](https://mooseframework.inl.gov/docs/PRs/20469/coverage_diff/framework.html)

[Full coverage report](https://mooseframework.inl.gov/docs/PRs/20469/coverage/framework)


## Modules coverage

Coverage did not change

## Full coverage reports

<details>
<summary>Reports</summary>

- [`framework`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/framework)
- [`chemical_reactions`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/chemical_reactions)
- [`combined`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/combined)
- [`contact`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/contact)
- [`external_petsc_solver`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/external_petsc_solver)
- [`fluid_properties`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/fluid_properties)
- [`fsi`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/fsi)
- [`functional_expansion_tools`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/functional_expansion_tools)
- [`geochemistry`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/geochemistry)
- [`heat_conduction`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/heat_conduction)
- [`level_set`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/level_set)
- [`misc`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/misc)
- [`navier_stokes`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/navier_stokes)
- [`peridynamics`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/peridynamics)
- [`phase_field`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/phase_field)
- [`porous_flow`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/porous_flow)
- [`ray_tracing`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/ray_tracing)
- [`rdg`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/rdg)
- [`reactor`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/reactor)
- [`richards`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/richards)
- [`stochastic_tools`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/stochastic_tools)
- [`tensor_mechanics`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/tensor_mechanics)
- [`thermal_hydraulics`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/thermal_hydraulics)
- [`xfem`](https://mooseframework.inl.gov/docs/PRs/20469/coverage/xfem)

</details>



This comment will be updated on new commits.

Commit:
Test conda-forge supplied mpich

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/985606/) on acd0c41 wanted to post the following:

The following file(s) are changed:

  conda/mpich/conda_build_config.yaml

The conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/985606/) on acd0c41 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/985606/) on acd0c41 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

milljm commented on conda/pyhit/meta.yaml:
```diff
@@ -1,4 +1,4 @@
-{% set build = 0 %}
+{% set build = 1 %}
 {% set version = "2022.01.24" %}
```
change version not build...

milljm commented on conda/mpich/build.sh:
```diff
@@ -1,104 +1,18 @@
 mkdir -p "${PREFIX}/etc/conda/activate.d" "${PREFIX}/etc/conda/deactivate.d"
 cat <<EOF > "${PREFIX}/etc/conda/activate.d/activate_${PKG_NAME}.sh"
-export CC=mpicc CXX=mpicxx FC=mpif90 F90=mpif90 F77=mpif77 C_INCLUDE_PATH=${PREFIX}/include MOOSE_NO_CODESIGN=true MPIHOME=${PREFIX} CXXFLAGS="$ACTIVATION_CXXFLAGS"
+export CC=mpicc CXX=mpicxx FC=mpif90 F90=mpif90 F77=mpif77 C_INCLUDE_PATH=${PREFIX}/include MOOSE_NO_CODESIGN=true MPIHOME=${PREFIX} CXXFLAGS="${ACTIVATION_CXXFLAGS}"
```
don't expand... (I would rather see this line not modified)

milljm commented:
Testing finished. Closing (for a rebase on devel without triggering a build)
